### PR TITLE
Remove docker and old integration tests cruft

### PR DESCRIPTION
**Pull request by bryevdv**
The is a trial PR to see about removing both the old selenium integration tests (that have not been run now for multiple years) as well as the docker CI bits (also not run for more than a year).

This is me officially throwing in the towel wrt to any docker related things in the main repo. If there is ever anyone who expresses interest in adding/maintaining anything for or around docker, I will propose to create a new separate repo for them to work on that independent of the main repo. 

It's posible there is yet more unused code under `tests/support`, I did not chase things too deep.

I also note the `PandasBokeh` downstream test is failing due to an import. I am not sure it is maintained any longer, we could probably jettison that as well.  (Edit: change added)

### CODE REVIEW

**Review by mattpap — APPROVED** [review]
I will propose to merge this as-is and remove any leftovers in another PR.

**Comment by bryevdv** [comment]
👍 marked as ready, I do at least want to mention at the meeting before merging

**Comment by bryevdv** [comment]
I'll plan to merge this EOD today unless anyone else expresses any comments/concerns

**Pull request merged**
